charts/csi-secret-provider-class/templates/_helpers.tpl

```diff
@@ -0,0 +1,63 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "csi-secret-provider-class.name" -}}
```

Comment:
Most of these are covered by `common` and are not used from here.

Reply:
11d0f3e759ba214811f4d382bfbac26083874c9b